tests/test_harness_notifications.py

```
@@ -39,13 +39,13 @@ def test_returns_multiple_keys(self) -> None:
         config.notify_slack_webhook = "https://hooks.slack.com/test"
         config.notify_discord_webhook = "https://discord.com/api/webhooks/test"
         config.notify_email = None
-        config.slack_bot_token = "xoxb-token"  # noqa: S105
+        config.slack_bot_token = "xoxb-token"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "slack_bot_token"

(S105)